LayoutTests/accessibility/aria-labelledby-stay-within.html

```diff
@@ -24,19 +24,24 @@
 </ul>
 
 </div>
-<p id="description"></p>
-<div id="console"></div>
 <script>
-        description("This tests that aria-labelledby does not append all sibling to an ARIA name");
+var output = "This tests that aria-labelledby does not append all sibling to an ARIA name\n\n";
 
-        if (window.accessibilityController) {
-            var labeledItem = document.getElementById("rep").focus();
-            var axButton = accessibilityController.focusedElement;
-            shouldBeEqualToString("platformValueForW3CName(axButton)", "Reply Item Five");
+if (window.accessibilityController) {
+    window.jsTestIsAsync = true;
```

Comment:
Does this test need to be async? I don't see any `await`s in it.

Reply:
Nope. At one point I thought we maybe discussed the idea of making all tests async because sooner or later many of them end up needing an await, but if you'd prefer we can wait.